### DISCUSSION TITLE
Singleton Auto-Refresh

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#14269] [Feature Requests]
Would be nice to set refresh rate for singleton tables.

Usecase: In order to get the new Flows working and trigering hooks/endpoints, I created a singleton table and read/update fields. In the Webapp it would be nice to see these changes with auto refresh

**Comment by azrikahar** [comment]
Although this isn't entirely the same thing as interval auto refresh, but we should be able to see changes to updated fields to content items (including singleton) once #14630 is resolved. Example:

https://user-images.githubusercontent.com/42867097/181431896-28bf5146-2ba1-4147-9856-ebda5429db69.mp4

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
